Assets/Scripts/Player/Movement/DashAbillity.cs

```diff
@@ -16,6 +19,7 @@ public class DashAbillity : MonoBehaviour
     
     [Header("Value")]
     [SerializeField] private float dashPower;
```

Comment:
```suggestion
    [SerializeField] private float dashSpeed;
```

Reply:
Noem het dashSpeed of dashDirection, want het is niet helemaal duidelijk wat je bedoel met dashPower en het lijkt erop alsof je het gebruikt voor de speed of voor de direction

Reply:
Verander die andere afkortingen dan ook

Reply:
keer som, like ding keer 2 ofzo